### PR TITLE
Move doc related stuff into folder

### PR DESCRIPTION
**Pull request by ryangjchandler**

### CODE REVIEW

**Comment by HugoDF** [comment]
Is it cool if we merge this? I would like to tackle #19 before we publish again cc @Te7a-Houdini

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # alpinejs-devtools
 
-<p align="center"><img width="720px" src="alpine-devtools-screenshot.png" alt="screenshot"></p>
+<p align="center"><img width="720px" src="docs/alpine-devtools-screenshot.png" alt="screenshot"></p>
```
Comment (HugoDF) [inline README.md:3]: total nitpick but any reason we're not using markdown's `` for this? @Te7a-Houdini

Comment (Te7a-Houdini) [reply README.md:3]: @HugoDF no reason we can use markdown for sure.

Comment (ryangjchandler) [reply README.md:3]: > total nitpick but any reason we're not using markdown's `` for this? @Te7a-Houdini

I did think this too, but kept it close to original. Will change now.

Comment (ryangjchandler) [reply README.md]: > total nitpick but any reason we're not using markdown's `` for this? @Te7a-Houdini

Doesn't look like GitHub understands the Markdown equivalent inside of a `<p>` tag, will leave as is.

**Pull request merged**